tagstudio/src/qt/flowlayout.py

```diff
@@ -23,101 +23,149 @@
 
 
 class FlowWidget(QWidget):
+    """
+    From what I understand:
+    A generic Qt window.
+    Please fix  my description
+    """
     def __init__(self, parent=None) -> None:
         super().__init__(parent)
         self.ignore_size: bool = False
 
 
 class FlowLayout(QLayout):
-    def __init__(self, parent=None):
+    """A generic Qt layout + the ability to keep track of the items in the layout."""
+    def __init__(self, parent=None) -> None:
+        # Initialize the parent class
         super().__init__(parent)
 
+        # In case parent is set
         if parent is not None:
+            # Set margins to 0.
             self.setContentsMargins(QMargins(0, 0, 0, 0))
 
-        self._item_list = []
-        self.grid_efficiency = False
+        # Initialize the items list to nothing.
+        self._item_list: list = []
+        # Initialize grid efficiency to "False"
+        self.grid_efficiency: bool = False
```

Comment:
This might be just me, but I dont see any value in these comments. The code here is pretty much self-explenatory and doesnt need any comment. Good comments should tell "why", not "what".

Reply:
Absolutly right. I  will get on that